step2/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM python:3.7
+
+ADD step1/requirements.txt /tmp
+RUN pip install -r /tmp/requirements.txt
+
+ADD step1/webconsole.py /
+
+# Add Tini
```

Comment:
We can ask participants and use Docker CE which has [tini built in](https://github.com/krallin/tini#using-tini)?

Reply:
NOTE: If you are using Docker 1.13 or greater, Tini is included in Docker itself. This includes all versions of Docker CE. To enable Tini, just pass the --init flag to docker run

I don't think we have access to that flag from minikube

Reply:
Cool. NP.